tests/plugins/test_plugin.py

```diff
@@ -16,6 +16,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+"""Module containing """
```

Comment:
Can you expand this description?

Reply:
Hey, sorry I must have gotten distracted in the middle of writing it 🙈 

I will fix it this weekend.

Reply:
@kik-kik any progress?